app/api/ddi-index-api/persons.js

```diff
@@ -33,19 +30,24 @@ const getPersons = async (filter) => {
 
   const searchParams = new URLSearchParams(Object.entries(value))
 
-  const payload = await get(`${personsEndpoint}?${searchParams.toString()}`, options)
+  const payload = await get(`${personsEndpoint}?${searchParams.toString()}`, user)
 
   return payload.persons
 }
 
-const getOrphanedOwners = async (filter = {}) => {
+/**
+ * @param user
+ * @param filter
+ * @return {Promise<import('./person.js').Person[]>}
+ */
+const getOrphanedOwners = async (user, filter = {}) => {
```

Comment:
Is this used in the enforcement UI, or is it just for the internal Portal?

Reply:
have left all of the api stuff in there, as would have been extra work to remove - and we don't know what is going to happen down the line.

Reply:
I've removed this as it's an admin task in the portal so no chance of it being done in the enforcement UI